specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json

```diff
@@ -28,6 +28,36 @@
     }
   },
   "paths": {
+    "/providers/Microsoft.HardwareSecurityModules/operations": {
+      "get": {
+        "tags": [
+          "DedicatedHsmOperations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Get a list of Dedicated HSM operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DedicatedHsmOperationListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a list of Dedicated HSM operations": {
+            "$ref": "./examples//DedicatedHsm_OperationsList.json"
```

Comment:
@lirenhe @KrisBash 

I created an example for this, however, I'm getting this error. I did further research and found that I might be missing a header field in my example but I can't spot it. It looks right to me.

message: 'Found errors in validating the response with statusCode "200" for x-ms-example "Get a list of Dedicated HSM operations" in operation "Operations_List".',
    innerErrors: [
      {
        code: 'INVALID_CONTENT_TYPE',
        message: 'Invalid Content-Type (application/octet-stream).  These are supported: ',
        path: []
      }
    ]
  },
  level: '\u001b[31merror\u001b[39m',
  message: 'Found errors in validating the response with statusCode "200" for x-ms-example "Get a list of Dedicated HSM operations" in operation "Operations_List".:\n'